article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -146,89 +159,106 @@ of the jacobian derivation see supplementary_notebook_2.ipynb). As an expansion
 the work done by Hoy and colleagues, we also allow users to use the Cholesky
 decomposition of the diffusion tensor to ensure that this is a positive definite tensor
 [@Koay2006-zo]. Due to increased mathematical complexity, the Cholesky decomposition
-is not used by default and it is not compatible with the analytical jacobian
+is not used by default and is not compatible with the analytical jacobian
 derivation.
 
 **Removing problematic estimates** For cases where the ground truth free water volume fraction is one (i.e. voxels
 containing only free water), the tissue's diffusion tensor component can erroneously
 overfit the free water diffusion signal and erroneously induce estimates of
-the water volume fraction near to zero. To remove these problematic cases, for all voxels with
-standard DTI mean diffusivity values larger than $2.7 \times 10^{-3} mm^{2}.s^{-1}$, the free
+the water volume fraction close to zero. To remove these problematic cases, for all voxels with
+standard DTI mean diffusivity values larger than $2.7 \times 10^{-3} mm^{2}.s^{-1}$, 
+
+stuff here
```

Comment:
Is that the MD of CSF? On what value is this threshold decided?  I have no idea, so just asking.